browser/scripts/site/accountpages.js

```diff
@@ -1 +1,92 @@
-// intentionally blank for future use
\ No newline at end of file
+var accountUI = function accountUI() {
```

Comment:
classes should have a capital first letter - this line could just read:

```
function AccountUI {
```

the assignment to a variable is redundant.